flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -120,6 +123,28 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     result.success("")
   }
 
+  private var didFetchAppStart = false
+
+  private fun fetchNativeAppStart(result: Result) {
+    val appStartTime = AppStartState.getInstance().getAppStartTime();
+    val isColdStart = AppStartState.getInstance().isColdStart();
+
+    if (appStartTime == null) {
+      result.error("1", "App start won't be sent due to missing appStartTime", null);
+    } else if (isColdStart == null) {
+      result.error("1", "App start won't be sent due to missing isColdStart", null);
+    } else {
+      val item = mapOf<String, Any?>(
+        "appStartTime" to appStartTime.getTime().toDouble(),
+        "isColdStart" to isColdStart,
+        "didFetchAppStart" to didFetchAppStart
+      )
+      result.success(item)
+    }
+    // This is always set to true, as we would only allow an app start fetch to happen once.
+    didFetchAppStart = true
```

Comment:
We could control that within the Dart side, so we don't even call the message channel in case we've sent the App start.